libpostal.yaml

```diff
@@ -0,0 +1,109 @@
+package:
+  name: libpostal
+  version: 1.1
+  epoch: 0
+  description: A C library for parsing/normalizing street addresses around the world. Powered by statistical NLP and open geo data-j8
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - file
+      - libtool
+      - mtdev-dev
+      - pkgconf-dev
+  environment:
+    CFLAGS: -Wno-incompatible-pointer-types # https://github.com/openvenues/libpostal/issues/677
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/openvenues/libpostal
+      expected-commit: 9c975972985b54491e756efd70e416f18ff97958
+      tag: v${{package.version}}
+
+  - runs: ./bootstrap.sh
+
+  - assertions:
+      required-steps: 1
+    pipeline:
+      - if: ${{build.arch}} == "x86_64"
+        uses: autoconf/configure
+        with:
+          opts: |
+            --disable-data-download
+      - if: ${{build.arch}} == "aarch64"
+        uses: autoconf/configure
+        with:
+          opts: |
+            --disable-sse2 \
+            --disable-data-download
+
+  - uses: autoconf/make
+
+  - uses: autoconf/make-install
+
+  - name: Create symlink for canonical share dir
+    runs: |
+      mkdir -p ${{targets.destdir}}/usr/share
+      ln -sf /usr/lib/libpostal ${{targets.destdir}}/usr/share/libpostal
+
+  - uses: strip
+
+subpackages:
+  - name: libpostal-dev
+    pipeline:
+      - uses: split/dev
+    dependencies:
+      runtime:
+        - libpostal
+    description: libpostal dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
+        - uses: test/tw/ldd-check
+
+update:
+  enabled: true
+  github:
+    identifier: openvenues/libpostal
+    strip-prefix: v
+    tag-filter: v
+    use-tag: true
+
+test:
+  environment:
+    contents:
+      packages:
+        - build-base
+        - libpostal-dev
+    environment:
+      LIBPOSTAL_DATA_DIR: /usr/share/libpostal
+  pipeline:
+    - name: Compile & run minimal parser
+      runs: |
+        cat > test_parse.c <<'EOF'
```

Comment:
Not a blocker, found this additional support test above https://github.com/openvenues/libpostal?tab=readme-ov-file#installation-windows section.
> `libpostal` has support for `pkg-config`, so you can use the `pkg-config` to print the flags needed to link your program against it:
> ```
> pkg-config --cflags libpostal         # print compiler flags
> pkg-config --libs libpostal           # print linker flags
> pkg-config --cflags --libs libpostal  # print both
> ```
> For example, if you write a program called app.c, you can compile it like this:
> ```
> gcc app.c `pkg-config --cflags --libs libpostal`